tests/testStacker.py

```diff
@@ -133,20 +133,20 @@ def testRequestMoreThanOneStat(self):
         """ Make sure we throw an exception if someone requests more than one type of statistics. """
 
         sctrl = afwMath.StatisticsControl()
-        imgList = afwImage.vectorImageF()
+        imgList = []
         for val in self.values:
             img = afwImage.ImageF(afwGeom.Extent2I(self.nX, self.nY), val)
-            imgList.push_back(img)
+            imgList.append(img)
 
         def tst():
-            afwMath.statisticsStack(imgList, afwMath.MEAN | afwMath.MEANCLIP, sctrl)
+            afwMath.statisticsStack(imgList, afwMath.Property(afwMath.MEAN | afwMath.MEANCLIP), sctrl)
```

Comment:
This is needed because `Property | Property` returns `int` and the function accepts a `Property`. There will be more places where this is needed. But there is no way around it. In pybind11 enums are just safer and sometimes require explicit conversions.